src/json_parse.rs

```diff
@@ -1,15 +1,15 @@
 use crate::db::{BotDbError, Repo};
-use crate::open_weather_map::weather::City;
-use serde_json::Value;
+use crate::open_weather_map::weather::CityDeserialize;
 use std::fs;
 
 pub async fn read_json_cities() -> Result<(), BotDbError> {
+    log::info!("Parsing cities from json");
     let repo = Repo::new().await?;
     let connection = repo.pool.get().await?;
 
     // check if cities are in db
     let n = connection
-        .execute("SELECT * FROM cities", &[])
+        .execute("SELECT * FROM cities LIMIT 1", &[])
```

Comment:
I think it should be moved to the repo

Reply:
Done in [8e9feff](https://github.com/pxp9/weather_bot_rust/pull/17/commits/8e9feff9b546089bf98bdedf9ba69874ff2cf740)